packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/simulate-ensemble-ciemss-operation.ts

```diff
@@ -28,7 +29,7 @@ export const normalValues = Object.freeze({
 });
 
 export interface SimulateEnsembleCiemssOperationState extends BaseState {
-	chartConfigs: string[][];
+	chartSettings: ChartSetting[] | null;
```

Comment:
@jryu01 what is the difference between "chartSettings" being null versus being an empty array?

Reply:
When it's null, we know the chartSettings hasn't been initialized at all (e.g. Right after the simulate/calibrate run). When it's empty, it tells that it might have been initialized already but there's no settings item currently. The settings can be empty when user deselects all existing charts.

Reply:
Do we have a real need to distinguish between the two different states or can they be  consolidated? 
One thing I would like to reduce is this type of disjunctive types because it feels like we are doing a lot of unnecessary null-checks.

Reply:
If we are fine with the behaviour we set the default charts on every run whenever there are no charts (even though they are cleared by user),  I think we can stick to `[]`

Reply:
alright, anyway, just some food for thought

Reply:
I would be happy to update this thought process such that we do not have `chartSettings[] | null` in all of our operator's states. I suspect that treating empty as a good enough reason to populate it should be fine from a user's perspective.
I think this should be done in a new PR as this one is already covering quite a lot of ground